### DISCUSSION TITLE
Active-active support for cluster

### DISCUSSION THREAD

**Discussion by idenkov** [valkey-io/valkey#605] [General]
Redis enterprise claims it support active-active replication when running as cluster. Is there any plans to support it in valkey?

**Comment by cjabrantes** [comment]
I was also looking for this and can't find it in documentation. keydb supports this, as we are using it.
Is this a lack in documentation or the the support is really missing? If missing, any plans to have such feature?

I would like to check/move to valkey, but that feature is needed.

Thanks!!
